### PR TITLE
Set up for e2e and 508 tests behind login component

### PR DESCRIPTION
**Pull request by jbalboni**
Related to #3605. 

This is the initial set up to write tests behind a login component for disability benefits.

One global change in here is the e2e test runner uses port 3333 so that we could use it to determine what environment we're running in and use the mock api appropriately.

Also, these are incomplete 508 and e2e tests. I just wanted to get the set up code out there and approved.

### CODE REVIEW

**Comment by jbalboni** [comment]
I'm not sure exactly who should review this, but FYI: @crwallace @alexose @ayaleloehr @akainic.

**Comment by alexose** [comment]
Looks great.  I like the idea of using `webstorage` to share the login information.

👍 👍 👍

File: test/accessibility/disability-benefits-508.spec.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const LoginHelpers = require('../util/login-helpers');
+
+module.exports = E2eHelpers.createE2eTest(
+  (client) => {
+    DisabilityHelpers.initClaimsListMock();
+
+    LoginHelpers.logIn(client, '/disability-benefits/track-claims', 3);
+
+    client
+      .url(`${E2eHelpers.baseUrl}/disability-benefits/track-claims`)
```
Comment (crwallace) [inline test/accessibility/disability-benefits-508.spec.js]: @jbalboni -- Is this your only route for the entire app?

**Review by crwallace — APPROVED** [review]
This looks good to me with one question. Would love for @jkassemi and @bshyong to check this out for awareness.

**Comment by bshyong** [comment]
LGTM thanks for the heads up on the port change

File: test/accessibility/disability-benefits-508.spec.js (hunk 1 above)
Comment (jbalboni) [inline test/accessibility/disability-benefits-508.spec.js]: No, there are others, I just wanted to get this set up code merged first.

Comment (crwallace) [inline test/accessibility/disability-benefits-508.spec.js]: Ok. Let's make sure that we have all of the routes before we check this off as done on the list.

**Pull request merged**